src/applications/check-in/locales/en/translation.json

```diff
@@ -292,6 +292,7 @@
   "learn-how-file-claims-other-expenses": "Learn how to file claims for other expenses",
   "set-up-direct-deposit": "Set up direct deposit",
   "set-up-direct-deposit-to-receive-travel-reimbursement": "You have to set up direct deposit to receive travel reimbursement. If you’ve already done this, no additional steps are needed.",
+  "file-a-travel-reimbursement-claim-if-youre-eligible": "File a travel reimbursement claim if you’re eligible for health care travel reimbursement. First, we need your last name and date of birth to make sure it’s you.",
```

Comment:
since this is all new stuff, you should go ahead and delete the old key and value this is replacing. You could have also just updated the previous value if you wanted to. You only need to follow the new key translation rules with translations that are already complete and are ordered in the list.

Reply:
These were all updates to existing pages, so the old values are still in use on the check-in travel pages (on purpose AFAIK)